src/app/components/Dialogs/TxDialog.tsx

```diff
@@ -79,10 +81,11 @@ export function TxDialog({ tx, onUserConfirmed, onSuccess }: Props) {
       {[TxStatus.PENDING, TxStatus.CONFIRMED, TxStatus.FAILED].includes(
         tx.status,
       ) && (
-        <>
-          <button data-close="" className="dialog-close" onClick={close}>
-            <span className="tw-sr-only">Close Dialog</span>
-          </button>
+        <ModalWrap>
```

Comment:
In the new standard to style the components, we prefer `module css` using `index.module.scss` file rather than using `styled-components`.
So, pls convert the styled components using `module CSS`.
Refer the file **_/app/pages/OriginsLaunchpad/pages/SalesDay/components/SaleSummary_**.
`module CSS` must leverage `tailwind CSS` as much as possible.
If  we can format a comment using tailwind CSS only, then we don't need to have it in module CSS, but will input them directly as `className` prop.